sopel/coretasks.py

```diff
@@ -599,6 +602,14 @@ def _parse_modes(bot, args):
             _send_who(bot, channel_name)
             return
 
+    if param_idx != len(params):
+        LOGGER.warning(
+            "%s extra arguments received for MODE: args=%r chanmodes=%r",
```

Comment:
```suggestion
            "%d extra arguments received for MODE: args=%r chanmodes=%r",
```

Reply:
Do we care about "1 extra argument**s**"?

Reply:
(addressed in another thread, so I could make a patch suggestion)